opentracing-spring-messaging-it/opentracing-spring-messaging-it-stream-common/pom.xml

```diff
@@ -33,7 +33,7 @@
   <dependencies>
     <dependency>
       <groupId>io.opentracing.contrib</groupId>
-      <artifactId>opentracing-spring-messaging</artifactId>
+      <artifactId>opentracing-spring-messaging-starter</artifactId>
```

Comment:
Why has this changed?

Reply:
Because there was no starter before. By adding the starter test application won't need to define interceptor bean.

Reply:
Oh I see, thanks